inputters/pandocast.lua

```diff
@@ -21,8 +21,8 @@ local utils = require("packages.markdown.utils")
 local function checkAstSemver(version)
   -- We shouldn't care the patch level.
   -- The Pandoc AST may change upon "minor" updates, though.
-  local major, minor = table.unpack(version)
-  local expected_major, expected_minor = table.unpack(Pandoc.API_VERSION)
+  local major, minor = pl.utils.unpack(version)
```

Comment:
There are subtle differences between `table.unpack` and `pl.utils.unpack` (which, last time I looked, called the former under the hood), but it doesn't seem we are in such a situation, are we?

Reply:
N.B. Not taken in #102, no side effect noted (well except #104 met when testing **pandocast**... but that's unrelated)